### DISCUSSION TITLE
[Feature Request]作者信息能否增加一张图片展示

### DISCUSSION THREAD

**Discussion by lurj7988** [vuepress-theme-hope/vuepress-theme-hope#3139] [Q&A] [answered]
### Checklist

- [X] I searched existing issues and no one else requests similar feature.
- [X] I think that 25%+ users are positive towards this feature.

### Describe the feature

可以是头像也可以是微信公众号二维码

**Comment by Mister-Hope** [comment, marked as answer]
作者信息的部分可以自定义头像，如果你需要在作者信息部分添加其他内容，可以使用组件替换功能。

- https://theme-hope.vuejs.press/zh/guide/blog/blogger.html#%E5%A4%B4%E5%83%8F%E5%92%8C%E5%8D%9A%E4%B8%BB%E5%90%8D%E7%A7%B0
- https://theme-hope.vuejs.press/zh/config/theme/feature.html#blog-avatar
- https://theme-hope.vuejs.press/zh/guide/advanced/replace.html

> - [x]  I think that 25%+ users are positive towards this feature.

另外我不是你的免费开发，请仔细评估自己的需求是否个人。
